docs/README.md

```diff
@@ -5,19 +5,23 @@ In this directory you will find files to assist with development and testing of
 the 5G-MAG Reference Tools 5GMS Application Server (AS).
 
 Files in this repository:
-- README.md      - Project README file.
-- LICENSE        - The software license for this project.
-- pyproject.toml - The Python project description for building and installing the application.
-- build_scripts/ - Scripts used when building the python project.
+- ATTRIBUTION_NOTICE - Project run-time attributions
+- LICENSE            - The software license for this project.
+- README.md          - Project README file.
+- pyproject.toml     - The Python project description for building and installing the application.
+- build_scripts/     - Scripts used when building the python project.
   - backend.py - build backend wrapper to trigger extra build actions.
-  - generate_openapi - Will generate the OpenAPI python modules if not already present.
-- docs/          - Development documentation and examples.
+  - generate_5gms_as_openapi - Will generate the OpenAPI python modules if not already present.
+- docs/              - Development documentation and examples.
   - README.md - This document.
   - chc.json - An example ContentHostingConfiguration JSON file which provisions a pull-ingest AS for "Big Buck Bunny".
```

Comment:
Shouldn't `chc.json` be removed now that the example is available from rt-common-shared/5gms?

Reply:
There's a file inside docs/ which is not on the list? example-application-server.conf
the json has been moved to external/rt-commom-shared/..., isn't it?

Reply:
These are the two things I've fixed directly:
https://github.com/5G-MAG/rt-5gms-application-server/issues/39#issuecomment-1283840836

Reply:
Absolutely